OneCommunityGlobal/HighestGoodNetworkApp
# Ramya lint fix  for Admin component

# Description
Lint fix for the Admin Component (src/component/Admin)


## Main changes explained:
-  Executed the link checker
-  Fixed changes manually  apart from fix done by lint--fix
-  Removed the component from .prettierignore and .eslintignore

## How to test:
- check into current branch
- do npm install and ... to run this PR locally
- Login as Admin
- go to Other Links->Popup Management
- check the app works as normal 
- set up ESLint and Prettier in VSCode
- check all files in src/component/Admin/** doesn't have errors
